headlamp-k8s/plugins
# prometheus: Add settings to prometheus plugin

This patch adds settings to the prometheus plugin
allowing the users to configure the service
address, timespan, enable metrics for each
cluster.

Fixes:
https://github.com/headlamp-k8s/plugins/issues/24

Includes the fix suggested here:
https://github.com/headlamp-k8s/plugins/issues/37#issuecomment-2230345145

Steps to Test:

1. Run prometheus plugin from the `prom_settings` branch.
2. Go to settings and configure the prometheus plugin, ie choose auto detect or provide service details to reach the prometheus service in the cluster.
3. Check if the configuration is working for the cluster by going to pod/namespace detail view.
